storage/google/cloud/storage/blob.py

```diff
@@ -379,6 +427,12 @@ def generate_signed_url(
         :returns: A signed URL you can use to access the resource
                   until expiration.
         """
+        if version is None:
+            version = "v2"
+            warnings.warn(DeprecationWarning(_SIGNED_URL_V2_DEFAULT_MESSAGE))
```

Comment:
Remove warning for now. We don't have a migration guide nor timeline for change. Two bits that are necessary to implement this warning.

Reply:
2d04844